xml/System.Composition/CompositionContext.xml

```diff
@@ -344,6 +352,7 @@ Note that this constructor is protected; it can only be called by classes derive
         <returns>
           <see langword="true" /> if an export was found; otherwise, <see langword="false" />.</returns>
         <remarks></remarks>
+        <exception cref="T:System.Composition.Hosting.CompositionFailedException"></exception>
```

Comment:
@maryamariyan this exception and the rest below are the same as the one you investigated, but I am not sure if I should simply copy-paste the same description you suggested. If you look at the exceptions above this one, the description is different. Can you please help determine which description to use for these?

Reply:
```suggestion
        <exception cref="T:System.Composition.Hosting.CompositionFailedException">No export was found for <paramref name="contract" />.</exception>
```